templates/default/web_app.conf.erb

```diff
@@ -39,7 +40,7 @@
   LogLevel info rewrite:trace1
   <%- else -%>
   LogLevel info
-  RewriteLog <%= node['apache']['log_dir'] %>/<%= @application_name %>-rewrite.log
+  RewriteLog <%= node['apache']['log_dir'] %>/<%= @params[:name] %>-rewrite.log
```

Comment:
we will not want to change this, as that is a `web_app` definition parameter that defaults to `@params[:name]`